test/nbrowser/DuplicateDocument.ts

```diff
@@ -7,6 +7,12 @@ describe("DuplicateDocument", function() {
   this.timeout(20000);
   const cleanup = setupTestSuite({team: true});
 
+  before(async function (){
```

Comment:
Perhaps we can add an inner describe block, `without external server`, and put this before hook in it. That way, some of the tests will still run even when an external server is being used, and we won't need to repeat the `skip` call in every test that doesn't work with an external server.